### PR TITLE
Jamie Fixes bugs and closes loopholes and Pisses off the Players and the Admins at the same time!

### PR DESCRIPTION
**Pull request by JamieD1**
Fixes: Achievements not opening
Fixes: Missing Admin verbs
Fixes: Collectable kitty ears not using colour
Fixes: Icon sprites on arcade machines
Fixes: Can sprites for monkey energy and cranberry.
Removes: Mods ability to grab from centcom (council approved)
Removes: Ability to RPD on Reebe (FUCKING TRANSPORT PIPES)
Removes: Stupid fun ending ion law (SUICIDE NOW KEK)
Removes: Ling Checks (Lings now Fakedeath on brain removal, (hopefully))

#### Changelog

:cl:  
bugfix: Achievements not opening
bugfix: Missing Admin verbs
bugfix: Collectable kitty ears not using colour
bugfix: Icon sprites on arcade machines
bugfix: Can sprites for monkey energy and cranberry.
rscdel: Mods ability to grab from centcom (council approved)
rscdel: Ability to RPD on Reebe (FUCKING TRANSPORT PIPES)
rscdel: Stupid fun ending ion law (SUICIDE NOW KEK)
rscdel: Ling Checks (Lings now Fakedeath on brain removal, (hopefully))
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Really impressed by that changelog

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1582,6 +1582,9 @@
 		var/atom/movable/AM = locate(href_list["admingetmovable"])
 		if(QDELETED(AM))
 			return
+		if(!check_rights(R_FUN))
+			if(AM.z == 1)
```
Comment (monster860) [inline code/modules/admin/topic.dm]: WOoP WOOP SHITCODE ALERT use `is_centcom_level(AM.z)` ree

also consider allowing teleporting from centcom to centcom.

**Review by monster860 — CHANGES_REQUESTED** [review]

**Comment by Hopekz** [comment]
watch out jcat might think this is a meme PR because you forgot to type out 2 paragraphs on why your fixes are good for the game!

**Comment by Jarod1200** [comment]
Make rpds shit out uncooked spaghetti if used on reebe

**Review by redmoogle — APPROVED** [review]
Changes are fine, looked at code

**Comment by boba247** [comment]
Is there a way mods can retrieve people from the admin prison?

**Comment by patpol4** [comment]
> Is there a way mods can retrieve people from the admin prison?

i think you just press the prison button again?

**Comment by Xoxeyos** [comment]
> 
> 
> > Is there a way mods can retrieve people from the admin prison?
> 
> i think you just press the prison button again?

That doesn't work.

**Pull request merged**